geomesa-convert/geomesa-convert-text/src/main/scala/org/locationtech/geomesa/convert/text/DelimitedTextConverter.scala

```diff
@@ -80,44 +76,16 @@ class DelimitedTextConverter(format: CSVFormat,
                              val validating: Boolean)
   extends LinesToSimpleFeatureConverter {
 
-  var curString: String = null
-  val q = Queues.newArrayBlockingQueue[String](32)
-  // if the record to write is bigger than the buffer size of the PipedReader
-  // then the writer will block until the reader reads data off of the pipe.
-  // For this reason, we have to separate the reading and writing into two
-  // threads
-  val writer = new PipedWriter()
-  val reader = new PipedReader(writer, options.pipeSize)  // record size
```

Comment:
I had talked with @cne1x about the best approach to fixing the issue, and we agreed that it would be best to remove the PipedWriter/Reader and go with the proposed line-by-line parsing for the short term. The catch is that this will be slower*, but it will not hang on bad formats. The plan is to work towards a more elegant solution that uses threading.

* most likely